src/main/resources/application.yaml

```diff
@@ -1,6 +1,6 @@
 quarkus:
   http:
-    port: 8080
+    port: 8081
```

Comment:
Why 8081? Which service is using 8080?

Reply:
@ligangty This is just for testing purpose, I set up 8080 for archive service in local, so setting 8081 for sidecar.
I'll change it back to 8080.